### PR TITLE
foundation: members

### PR DESCRIPTION
**Pull request by reez**
Adding two new pages:
- members
- become-a-member

Content is just placeholder for now.

### CODE REVIEW

**Comment by reez** [comment]
Added BDKF membership pdf file and link

File: docs/.vuepress/public/membership-overview-deck.pdf
Comment (notmandatory) [inline docs/.vuepress/public/membership-overview-deck.pdf:1]: Let's remove this. I'd rather just leave the email link below and follow up and discuss membership individually with any companies interested before sending the form for them to apply.

File: docs/foundation/become-a-member.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+lastUpdated: false
+---
+
+# Become a Member
+
+The Bitcoin Dev Kit Foundation is a non-profit organization dedicated to supporting the development of Bitcoin infrastructure. We offer corporate membership tiers: Platinum, Gold, and Silver. We also provide an Associate Membership tier for non-profit organizations.
+
+**We do not currently provide memberships for individuals.**
+
+Membership benefits include representation in the Foundation and influence on Bitcoin Dev Kit projects. Download our Membership Overview Deck to learn more.
```
Comment (notmandatory) [inline docs/foundation/become-a-member.md]:
```suggestion
Membership benefits include listing on the BDK Foundation members page and representation in the Foundation board of directors. 
```

File: docs/foundation/become-a-member.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+<div style="text-align: left; margin-bottom: 2rem;">
+  <a href="/membership-overview-deck.pdf" style="display: inline-block; background-color: var(--docs-primary-dark); color: var(--docs-dark); padding: 10px 20px; text-decoration: none; border-radius: 5px; font-weight: bold;">Membership Overview Deck</a>
+</div>
```
Comment (notmandatory) [inline docs/foundation/become-a-member.md]:
```suggestion
```

File: docs/foundation/become-a-member.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+_* Associate membership is available only to nonprofits & educational institutions, and not to individuals. Acceptance is subject to board approval._
+
+### Ready to apply?
+
+<div style="text-align: left; margin-top: 2rem; margin-bottom: 2rem;">
+  <a href="mailto:hello@bitcoindevkit.org" style="display: inline-block; background-color: var(--docs-primary-dark); color: var(--docs-dark); padding: 10px 20px; text-decoration: none; border-radius: 5px; font-weight: bold;">Send us an email</a>
+</div>
+
+The Bitcoin Dev Kit project is proudly supported by a number of members. If you'd like to become a member please reach out at: [hello@bitcoindevkit.org](mailto:hello@bitcoindevkit.org)
```
Comment (notmandatory) [inline docs/foundation/become-a-member.md]:
```suggestion
The Bitcoin Dev Kit project is made possible through the support of the BDK Foundation and its members. If you'd like to become a member please reach out at: [hello@bitcoindevkit.org](mailto:hello@bitcoindevkit.org)
```

File: docs/foundation/become-a-member.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+You can also download a sample copy of our Membership Agreement terms. <!-- Add link when available -->
```
Comment (notmandatory) [inline docs/foundation/become-a-member.md]: This should also be removed.
```suggestion
```

File: docs/foundation/members.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+sidebar: true
+tagline: "Foundation"
+description: "Information about the Bitcoin Dev Kit Foundation"
+editLink: false
+lastUpdated: false
+---
+
+# Members
+
+The Bitcoin Dev Kit project is proudly supported by a number of members.
```
Comment (notmandatory) [inline docs/foundation/members.md]:
```suggestion
The Bitcoin Dev Kit project is proudly supported by the Bitcoin Dev Kit Foundation and its members.
```

File: docs/foundation/members.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+<div class="members">
+  <div class="row">
+      <a href="https://spiral.xyz/" target="_blank" rel="nofollow noopener">
+          <img src="/img/spiral.svg" alt="Sponsor Spiral">
+          Spiral
+      </a>
+      <a href="https://opensats.org/"  target="_blank" rel="nofollow noopener">
+        <img src="/img/opensats.png" alt="OpenSats">
+        OpenSats
+      </a>
```
Comment (notmandatory) [inline docs/foundation/members.md]: These are already listed a platinum and gold so shouldn't be listed under silver. I suspect you just put these here to test the formatting. I do like how it looks running horizontally.

File: docs/foundation/members.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+<div style="text-align: left; margin-bottom: 2rem;">
+  <a href="/foundation/become-a-member" style="display: inline-block; background-color: var(--docs-primary-dark); color: var(--docs-dark); padding: 10px 20px; text-decoration: none; border-radius: 5px; font-weight: bold;">Become A Member</a>
+</div>
+
+## Platinum
+
+<div class="members">
+  <div class="row">
+      <a href="https://spiral.xyz/" target="_blank" rel="nofollow noopener">
+          <img src="/img/spiral.svg" alt="Sponsor Spiral">
```
Comment (notmandatory) [inline docs/foundation/members.md]:
```suggestion
          <img src="/img/spiral.svg" alt="Spiral">
```

**Review by notmandatory — COMMENTED** [review]
Overall looks great. I have a few comments.

File: docs/.vuepress/public/membership-overview-deck.pdf
Comment (reez) [reply docs/.vuepress/public/membership-overview-deck.pdf:1]: Removed

File: docs/foundation/become-a-member.md (hunk 1 above)
Comment (reez) [reply docs/foundation/become-a-member.md]: Updated

File: docs/foundation/become-a-member.md (hunk 2 above)
Comment (reez) [reply docs/foundation/become-a-member.md]: Removed.

File: docs/foundation/become-a-member.md (hunk 3 above)
Comment (reez) [reply docs/foundation/become-a-member.md]: Updated.

File: docs/foundation/become-a-member.md (hunk 4 above)
Comment (reez) [reply docs/foundation/become-a-member.md]: Removed.

File: docs/foundation/members.md (hunk 5 above)
Comment (reez) [reply docs/foundation/members.md]: Updated.

File: docs/foundation/members.md (hunk 7 above)
Comment (reez) [reply docs/foundation/members.md]: Good catch, updated.

File: docs/foundation/members.md (hunk 6 above)
Comment (reez) [reply docs/foundation/members.md]: Updated.

**Comment by reez** [comment]
I think I resolved all the comments, and then I just rebased on `master`

**Review by notmandatory — APPROVED** [review]
ACK 1307ed79c459713ddbc863f4982586d07d3f3398

Thanks !

**Pull request merged**